modules/Bio/EnsEMBL/Compara/PipeConfig/Plants/RiceCultivarsProteinTrees_conf.pm

```diff
@@ -0,0 +1,85 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::Plants::RiceCultivarsProteinTrees_conf
+
+=head1 SYNOPSIS
+
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Plants::RiceCultivarsProteinTrees_conf -host mysql-ens-compara-prod-X -port XXXX
+
+=head1 DESCRIPTION
+
+The Rice cultivars PipeConfig file for CultivarsProteinTrees pipeline that should automate most of the pre-execution tasks.
+
+=cut
+
+package Bio::EnsEMBL::Compara::PipeConfig::Plants::RiceCultivarsProteinTrees_conf;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Compara::PipeConfig::Plants::CultivarsProteinTrees_conf');
+
+
+sub default_options {
+    my ($self) = @_;
+
+    return {
+        %{$self->SUPER::default_options},
+
+        # Parameters to allow merging different runs of the pipeline
+        'collection'       => 'rice_cultivars',  # The name of the species-set within that division
+        'dbID_range_index' => 30,
+        'label_prefix'     => 'rice_cultivars',
```

Comment:
I just have the one suggestion, to add the underscore to the end of the prefix, similar to other collections (e.g. `wheat_cultivars_`):
```suggestion
        'label_prefix'     => 'rice_cultivars_',
```